docs/development/module/tests.md

```diff
@@ -6,104 +6,116 @@ Runtime tests allow you to verify the logic in your runtime module by mocking a
 
 ## Unit Testing
 
-Substrate uses the existing [unit testing](https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html) framework provided by Rust.
+Substrate uses the existing [unit testing](https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html) framework provided by Rust. To run tests, the command is 
 
-The average runtime test will look like:
-
-```rust
-#[test]
-fn my_runtime_test() {
-	with_externalities(&mut new_test_ext(), || {
-		some_assertion!(...);
-	});
-}
+```bash
+cargo test <optional: test_name>
 ```
 
 ## Mock Runtime Environment
 
-To support the specific needs of testing a Substrate runtime, you need to construct a "mock" runtime environment. This involves importing the logic of other runtime modules, such as the System module, and establishing runtime storage.
+To test a Substrate runtime, construct a mock runtime environment. The [substrate recipes](https://substrate.dev/recipes/testing/mock.html) provides a simple example of using a mocked runtime to test module logic. For more examples, see the `mock.rs` and `test.rs` files in most substrate modules.
 
-In general, we suggest looking at and mimicking the various examples of runtime tests in the Substrate Runtime Module Library (SRML). This document won't be as comprehensive as what is provided in our SRML.
+### Mock Runtime Storage
 
-### Mock Runtime Modules
+The [`runtime-io`](https://crates.parity.io/sr_io/index.html#enums) crate exposes a [`TestExternalities`](https://crates.parity.io/sr_io/type.TestExternalities.html) implementation frequently used for mocking storage in tests. It is the type alias for an in-memory, hashmap-based externalities implementation in [`substrate_state_machine`](https://crates.parity.io/substrate_state_machine/index.html)] referred to as [`TestExternalities`](https://crates.parity.io/substrate_state_machine/struct.TestExternalities.html).
 
-To test your module, you will need to construct a configuration type (`Test`) which implements each of the configuration traits of modules you want to use.
+In the [basic mock runtime's recipe](https://substrate.dev/recipes/testing/mock.html), an `ExtBuilder` object is defined to build an instance of [`TestExternalities`](https://crates.parity.io/sr_io/type.TestExternalities.html).
```

Comment:
```suggestion
In the [basic mock runtime's recipe](https://substrate.dev/recipes/testing/mock.html), an `ExtBuilder` object is defined to build an instance of [`TestExternalities`](https://substrate.dev/rustdocs/master/sr_io/type.TestExternalities.html).
```

@shawntabrizi asked me the other day to start using the substrate.dev links for rustdocs.

Reply:
good to know